i18n/en-US.properties

```diff
@@ -1136,6 +1136,8 @@ boxui.securityControls.webDownloadExternalOwnersEditors = Download restricted on
 boxui.securityControls.webDownloadOwners = Download restricted on web, except Owners/Co-Owners.
 # Bullet point that summarizes web download restrictions applied to classification, when restriction applies to managed users except Owners/Co-Owners/Editors
 boxui.securityControls.webDownloadOwnersEditors = Download restricted on web, except Owners/Co-Owners/Editors.
+# text shown on the Clear All option in the options list
+boxui.select-field.clearAll = Clear All
```

Comment:
Let's use camel case instead of `-` to be consistent. Could you change it to `boxui.selectField.clearAll`?